Tasks/AzureAppServiceManage/operations/KuduServiceUtils.ts

```diff
@@ -128,7 +129,7 @@ export class KuduServiceUtils {
     }
 
     private _getExtensionLocalPath(extensionInfo: JSON): string {
-        var extensionId: string = extensionInfo['id'];
+        var extensionId: string = extensionInfo['id'].replace(pythonExtensionPrefix, "");
```

Comment:
do you want to do this only for python ?

Reply:
yes, planned to do only for Python. but the check wont harm other extensions, as they dont have this prefix as of now. will be removing this once dynamic extension feature is available. //adding comment to ensure the same.